### ISSUE TITLE
Dynmap looked crazzy now in 8.0.0[BUG] 

### ISSUE THREAD

**Issue by CloudeLecaw** [FabioZumbi12/RedProtect#821]
The settings in config seems not to not reaci to the dynmap.. in result the dynma look a bit crazy

![Snag_a97e368](https://user-images.githubusercontent.com/12080017/174483341-8fa3fcd9-4809-4f00-a1c5-d850cca358d5.png)

**Comment by Blueeyestar** [comment]
I can only confirm this. I was able to duplicate this error. Even on a newly generated RedProtect folder.
Makers cannot be issued or the boxes with the names can be deactivated.
You can't even change the name of the RedProtect. No matter what you write, the name RedProtect always remains

# Group name to show on hide/show tab map.
        marks-groupname=Zone
        min-zoom=0
        # Show icons under regions.
        show-icon=false
        # Show names under regions.
        show-label=false
        # Show leaders and admins on hover?
        show-leaders-admins=true
![redp](https://user-images.githubusercontent.com/35557302/174963957-659864c2-45da-4aab-9713-9ad90e45fedb.png)

**Comment by FabioZumbi12** [comment]
@CloudeLecaw i don't know whats exactly its wrong just looking to your map. What's you trying and whats redprotect is not doing? @Blueeyestar i think i got it.

**Comment by CloudeLecaw** [comment]
look here this is how the Dynmap looks with Version 7.7.3 Build 413
And tis is how it should look. You see the difference?

![image](https://user-images.githubusercontent.com/12080017/177212388-8f03e4dc-c8e2-48bb-b537-82ae2d39db9e.png)

**Comment by CloudeLecaw** [comment]
Maybe a new Information..
With out Old Config.. all loads fine  in 7.7.3

When i let a NEW config generate in 7.7.3 we also have this wrong Maplook
older file
https://www.toptal.com/developers/hastebin/dimolurefu.sql


So it seems liek something in the config with the dynap settings may went wrong?
Please take a look at it
new file:
https://www.toptal.com/developers/hastebin/tozufocebe.coffeescript


pleae try out both and tell us why the old works like wanted..
and the new looks so ...ugly.. on dynmap

**Comment by Blueeyestar** [comment]
It's nice that there's an update for 1.19.2 since October 22, 2022, but RedProtect is still not usable with the dynmap. Apparently you don't understand where the problem lies or is it so defective with the update that you can't fix it anymore?

Unfortunately, the RedProtect can no longer be used since 8.0.0 if you also use the dynmap.

![redprotect](https://user-images.githubusercontent.com/35557302/197584673-b6940275-13d0-4edd-89e6-e4e052452c13.png)

**Comment by CloudeLecaw** [comment]
any updateds here?

We really wanne update to version 8.0.0...

we tested Build #417 today...the issue still exists..
we want the map back looking like this
![image](https://user-images.githubusercontent.com/12080017/177212388-8f03e4dc-c8e2-48bb-b537-82ae2d39db9e.png)

and NOT like this
![image](https://user-images.githubusercontent.com/12080017/177213326-6a74e873-9ee5-492a-87f0-ea7782aaa4ff.png

**Comment by Blueeyestar** [comment]
It probably doesn't interest anyone that there are problems and errors....

**Comment by Blueeyestar** [comment]
too bad even with Build #418 the garbage is still not fixed.....why is there an update if the problem remains the same....very very very unfortunate...

![image](https://user-images.githubusercontent.com/35557302/208480531-69d50785-1dbd-479a-9855-bac6c66fc8c7.png)

**Comment by Blueeyestar** [comment]
Since unfortunately no answer came from the developer, I had to start solving the problem myself.
So I came up with the idea of looking for the dynmap files from an old working RP version. I then exchanged these for the defective files in the current RedProtect-8.0.0-SNAPSHOT-b418-Universal. And then the dynmap works again in RP as you are used to

Old dynmap version RedProtect_7.7.3-Build413:
![rpalt](https://user-images.githubusercontent.com/35557302/226905852-8f6d61d1-dbca-4f10-a119-b1c20001eade.png)

New broken dynmap version RedProtect-8.0.0-SNAPSHOT-b418-Universal:
![rpdefekt](https://user-images.githubusercontent.com/35557302/226906070-3d62ea46-48e9-49da-8878-e769b41c05b4.png)

You can download the RedProtect-8.0.0-SNAPSHOT-b418-Universal with working Dynmap here:
[RedProtect-8.0.0.SNAPSHOT-b418-Universal.zip](https://github.com/FabioZumbi12/RedProtect/files/11039961/RedProtect-8.0.0.SNAPSHOT-b418-Universal.zip)

The dynmap is now running in the new version.
But there is still a bug in the version that was reported in June 2022. Therefore, this is not recommended because this bug is very annoying.

https://github.com/FabioZumbi12/RedProtect/issues/820

That's why I've adapted the working RP version: RedProtect_7.7.3-Build413 to 1.19 and added the mobs and animal heads so that they don't have a placeholder in the menu. The version also works perfectly in Minecraft version 1.19.4.

[RedProtect_7.7.3-Build413.zip](https://github.com/FabioZumbi12/RedProtect/files/11039991/RedProtect_7.7.3-Build413.zip)

We'll keep looking to fix some issues but would be really grateful if FabioZumbi12 gets active again and also fixes the "Enter Region in chat in Redprotect" issue as we don't yet know which file is causing this issue.

**Comment by CloudeLecaw** [comment]
If you want the Addoons Killer Projectiles and RentRegion for 7.7.3 here they are.
i sendet them to you here, because on the Redprotect Github you only get the once for Version 8 anymore.
Dont ask me for other Addons of it for 7.7.3 i have no others because we didnt used the others.
[Redprotect_7.7.3-Addons_KillerProjectilesAndRentRegion.zip](https://github.com/FabioZumbi12/RedProtect/files/11146248/Redprotect_7.7.3-Addons_KillerProjectilesAndRentRegion.zip)

**Comment by FabioZumbi12** [comment]
Hi.
Try this version. Its a 8.1.0 created directly from the last 7.7.3
[RedProtect-8.1.0-SNAPSHOT-Universal.zip](https://github.com/FabioZumbi12/RedProtect/files/11186693/RedProtect-8.1.0-SNAPSHOT-Universal.zip)

**Comment by Blueeyestar** [comment]
> Hi. Try this version. Its a 8.1.0 created directly from the last 7.7.3 but with all latest changes except changes on configurations, and added an option:
> 
> ```yaml
> # Convert all regions with minY = 0 to the world min y? Fix existing regions not protecting below 0 on 1.19+!
> convert-zeros-y=true
> ```
> 
> [RedProtect-8.1.0-SNAPSHOT-Spigot.zip](https://github.com/FabioZumbi12/RedProtect/files/11188044/RedProtect-8.1.0-SNAPSHOT-Spigot.zip)

Is this a trailing April Fool's joke?
You throw out a new RedProtect version 8.1.0 without correcting the serious bugs in the version?
Instead, are they continued?

Did you actually take a look at this thread?
Neither the Dynmap problem I solved here nor the Enter Region in chat problem have been fixed.
The 8.1.0 is not usable!!!!

**Comment by CloudeLecaw** [comment]
well that really was a bit disapointed.. im not sure why the version 8.1. was made when its results in the same issue as before. cant you simply use the 7.7.3 versions and do the snaller changes on this working on?

**Comment by FabioZumbi12** [comment]
> Since unfortunately no answer came from the developer, I had to start solving the problem myself. So I came up with the idea of looking for the dynmap files from an old working RP version. I then exchanged these for the defective files in the current RedProtect-8.0.0-SNAPSHOT-b418-Universal. And then the dynmap works again in RP as you are used to
> 
> Old dynmap version RedProtect_7.7.3-Build413: ![rpalt](https://user-images.githubusercontent.com/35557302/226905852-8f6d61d1-dbca-4f10-a119-b1c20001eade.png)
> 
> New broken dynmap version RedProtect-8.0.0-SNAPSHOT-b418-Universal: ![rpdefekt](https://user-images.githubusercontent.com/35557302/226906070-3d62ea46-48e9-49da-8878-e769b41c05b4.png)
> 
> You can download the RedProtect-8.0.0-SNAPSHOT-b418-Universal with working Dynmap here: [RedProtect-8.0.0.SNAPSHOT-b418-Universal.zip](https://github.com/FabioZumbi12/RedProtect/files/11039961/RedProtect-8.0.0.SNAPSHOT-b418-Universal.zip)
> 
> **As you can see, 2 dynmap .class files are missing in the new RP version. These were added from the old one. Then the dynmap worked again in the 8.0.0 version**
> 
> The dynmap is now running in the new version. But there is still a bug in the version that was reported in June 2022. Therefore, this is not recommended because this bug is very annoying.
> 
> #820
> 
> That's why I've adapted the working RP version: RedProtect_7.7.3-Build413 to 1.19 and added the mobs and animal heads so that they don't have a placeholder in the menu. The version also works perfectly in Minecraft version 1.19.4.
> 
> [RedProtect_7.7.3-Build413.zip](https://github.com/FabioZumbi12/RedProtect/files/11040123/RedProtect_7.7.3-Build413.zip)
> 
> We'll keep looking to fix some issues but would be really grateful if FabioZumbi12 gets active again and also fixes the "Enter Region in chat in Redprotect #820" issue as we don't yet know which file is causing this issue.

Yeah, this version is based on the last 7.7.3 that you used to build this version. Instead to continue the 8.0.0 i just jumped from 7.7.3-Build413 to 8.1.0. with the last bugfixes present here:
https://github.com/FabioZumbi12/RedProtect/compare/master-legacy1.18...RedProtect-8.0.0

I got the 8.0.0 updated a lot of things related to configuration and bugged a lot of things like dynmap and redprotect configs too.
Running here locally with 8.1.0 i didnt got the issues i got on 8.0.0

**Comment by Blueeyestar** [comment]
> RedProtect_7.7.3-Build413.zip

Are you sure that you also took the RedProtect_7.7.3-Build413.zip and not accidentally the 8.0.0?

Because I just set up a server from scratch.
And then Red Protect installed from scratch.

Unfortunately, the errors that are present in the 8.0.0 version are still the same!!!

You can use Show messages on enter, exit and welcome messages. it doesn't matter how you change it. RP always uses chat function.
![2023-04-10_20 36 19](https://user-images.githubusercontent.com/35557302/230970233-613543ed-c15f-4cf3-8b10-08babda0f905.png)
![bossbar](https://user-images.githubusercontent.com/35557302/230970244-4258b250-fc7f-4394-a804-03b70376efc1.png)




It is still not adopted if you change the map name in the config: Group name to show on hide/show tab map

No matter which name you choose, it always remains Redprotect.

![zonen](https://user-images.githubusercontent.com/35557302/230970280-ad1038d2-08e6-49bd-bf24-0876be8aea23.png)

![rp](https://user-images.githubusercontent.com/35557302/230970291-5227234d-d513-4a32-8cce-33aeb68da149.png)

The dynmap is still defective in appearance.

The problem is the same as in the 8.0.0 version.

**Comment by Blueeyestar** [comment]
> > Since unfortunately no answer came from the developer, I had to start solving the problem myself. So I came up with the idea of looking for the dynmap files from an old working RP version. I then exchanged these for the defective files in the current RedProtect-8.0.0-SNAPSHOT-b418-Universal. And then the dynmap works again in RP as you are used to
> > Old dynmap version RedProtect_7.7.3-Build413: ![rpalt](https://user-images.githubusercontent.com/35557302/226905852-8f6d61d1-dbca-4f10-a119-b1c20001eade.png)
> > New broken dynmap version RedProtect-8.0.0-SNAPSHOT-b418-Universal: ![rpdefekt](https://user-images.githubusercontent.com/35557302/226906070-3d62ea46-48e9-49da-8878-e769b41c05b4.png)
> > You can download the RedProtect-8.0.0-SNAPSHOT-b418-Universal with working Dynmap here: [RedProtect-8.0.0.SNAPSHOT-b418-Universal.zip](https://github.com/FabioZumbi12/RedProtect/files/11039961/RedProtect-8.0.0.SNAPSHOT-b418-Universal.zip)
> > **As you can see, 2 dynmap .class files are missing in the new RP version. These were added from the old one. Then the dynmap worked again in the 8.0.0 version**
> > The dynmap is now running in the new version. But there is still a bug in the version that was reported in June 2022. Therefore, this is not recommended because this bug is very annoying.
> > #820
> > That's why I've adapted the working RP version: RedProtect_7.7.3-Build413 to 1.19 and added the mobs and animal heads so that they don't have a placeholder in the menu. The version also works perfectly in Minecraft version 1.19.4.
> > [RedProtect_7.7.3-Build413.zip](https://github.com/FabioZumbi12/RedProtect/files/11040123/RedProtect_7.7.3-Build413.zip)
> > We'll keep looking to fix some issues but would be really grateful if FabioZumbi12 gets active again and also fixes the "Enter Region in chat in Redprotect #820" issue as we don't yet know which file is causing this issue.
> 
> Yeah, this version is based on the last 7.7.3 that you used to build this version. Instead to continue the 8.0.0 i just jumped from 7.7.3-Build413 to 8.1.0. with the last bugfixes present here: [master-legacy1.18...RedProtect-8.0.0](https://github.com/FabioZumbi12/RedProtect/compare/master-legacy1.18...RedProtect-8.0.0)
> 
> I got the 8.0.0 updated a lot of things related to configuration and bugged a lot of things like dynmap and redprotect configs too. Running here locally with 8.1.0 i didnt got the issues i got on 8.0.0

Are you sure that you also took the RedProtect_7.7.3-Build413.zip and not accidentally the 8.0.0?

Because I just set up a server from scratch.
And then Red Protect installed from scratch.

Unfortunately, the errors that are present in the 8.0.0 version are still the same!!!

You can use Show messages on enter, exit and welcome messages. it doesn't matter how you change it. RP always uses chat function.
![2023-04-10_20 36 19](https://user-images.githubusercontent.com/35557302/230970233-613543ed-c15f-4cf3-8b10-08babda0f905.png)
![bossbar](https://user-images.githubusercontent.com/35557302/230970244-4258b250-fc7f-4394-a804-03b70376efc1.png)




It is still not adopted if you change the map name in the config: Group name to show on hide/show tab map

No matter which name you choose, it always remains Redprotect.

![zonen](https://user-images.githubusercontent.com/35557302/230970280-ad1038d2-08e6-49bd-bf24-0876be8aea23.png)

![rp](https://user-images.githubusercontent.com/35557302/230970291-5227234d-d513-4a32-8cce-33aeb68da149.png)

The dynmap is still defective in appearance.

The problem is the same as in the 8.0.0 version.

**Comment by Blueeyestar** [comment]
if you took the RedProtect_7.7.3-Build413 why are there only 4 dynmap files in the 8.10 version?

In the RedProtect_7.7.3-Build413 there are 6 dynmap files. The dynmap only works if the other 2 files are included.

But these are missing in the 8.1.0 version just like in the 8.0.0 version
![dynmap](https://user-images.githubusercontent.com/35557302/230972752-d9fefac9-3aca-4172-bedc-3158ad6a7611.png)

**Comment by FabioZumbi12** [comment]
What the paths of that 2 other classes? Here on dev side, i have only one class called DynmapHook.class. Maybe the problem is on build way or similar.

**Comment by FabioZumbi12** [comment]
My God, i got that my IDE its trolling me. Checkout this version:

[RedProtect-8.1.0-SNAPSHOT-Spigot.zip](https://github.com/FabioZumbi12/RedProtect/files/11193489/RedProtect-8.1.0-SNAPSHOT-Spigot.zip)

**Comment by Blueeyestar** [comment]
> My God, i got that my IDE its trolling me. Checkout this version:

Congratulations now works the 8.1.0

great job :)

**Comment by Blueeyestar** [comment]
It is best to upload the working version here http://host.areaz12server.net.br:8081/job/RedProtect/  

and with Spigot so that everyone downloads the working version and not the next one comes and says it doesn't work

**Comment by CloudeLecaw** [comment]
if you update the new 8.1.0 pls dont forget to also update the Addons @FabioZumbi12 <3 :§   i know it a bit out of theme :3

```
[22:11:37] [Server thread/ERROR]: Error occurred while enabling KillerProjectiles v8.0.0-SNAPSHOT (Is it up to date?)
java.lang.NoSuchMethodError: 'void br.net.fabiozumbi12.RedProtect.Bukkit.API.RedProtectAPI.addFlag(java.lang.String, boolean, boolean)'
	at br.net.fabiozumbi12.killerprojectiles.KillerProjectiles.onEnable(KillerProjectiles.java:53) ~[RedProtect_8.0.0-Build418-RPKillerProjectiles.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:279) ~[paper-api-1.19.4-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[paper-1.19.4.jar:git-Paper-503]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.19.4.jar:git-Paper-503]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.19.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugin(CraftServer.java:555) ~[paper-1.19.4.jar:git-Paper-503]
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugins(CraftServer.java:466) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:638) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:437) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1104) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper-1.19.4.jar:git-Paper-503]
	at java.lang.Thread.run(Thread.java:1623) ~[?:?]
[22:11:37] [Server thread/INFO]: [KillerProjectiles] Disabling KillerProjectiles v8.0.0-SNAPSHOT
[22:11:37] [Server thread/ERROR]: Error occurred (in the plugin loader) while disabling KillerProjectiles v8.0.0-SNAPSHOT (Is it up to date?)
java.lang.NoSuchMethodError: 'void br.net.fabiozumbi12.RedProtect.Bukkit.API.RedProtectAPI.removeFlag(java.lang.String, boolean)'
	at br.net.fabiozumbi12.killerprojectiles.KillerProjectiles.onDisable(KillerProjectiles.java:77) ~[RedProtect_8.0.0-Build418-RPKillerProjectiles.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[paper-api-1.19.4-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.disablePlugin(PaperPluginInstanceManager.java:224) ~[paper-1.19.4.jar:git-Paper-503]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.disablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.19.4.jar:git-Paper-503]
	at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:537) ~[paper-api-1.19.4-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:193) ~[paper-1.19.4.jar:git-Paper-503]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.19.4.jar:git-Paper-503]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.19.4-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugin(CraftServer.java:555) ~[paper-1.19.4.jar:git-Paper-503]
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugins(CraftServer.java:466) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:638) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:437) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1104) ~[paper-1.19.4.jar:git-Paper-503]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper-1.19.4.jar:git-Paper-503]
	at java.lang.Thread.run(Thread.java:1623) ~[?:?]
```

**Comment by FabioZumbi12** [comment]
Just here for now

[RedBackups-8.1.0-SNAPSHOT.zip](https://github.com/FabioZumbi12/RedProtect/files/11193790/RedBackups-8.1.0-SNAPSHOT.zip)

**Comment by CloudeLecaw** [comment]
The Problem is solved with thie new 8.1.0 Build 419. So the issue can be closed <3
Thank you for solving this for us.
